exercise_0/main.cpp

```diff
@@ -36,9 +33,9 @@ int     words_checker(std::istream &read_file, char const *stop) {
     consonants = 0;
```

Comment:
Shorter:
int vowels = 0;
int consonants = 0;

Reply:
Yes, I know. It's bad style to write like that?

Reply:
No, it's not "bad". But, as long as we do not have pay for each line of code, I do not see why write it in this way.

Reply:
Maybe someday:) For me, it's just more readable.

Reply:
If so, I'm OK with it.